.github/workflows/terraform.yaml

```diff
@@ -1,22 +1,59 @@
 name: Validate Terraform
+
 on:
+  - push
   - pull_request
+
 jobs:
   terraform:
     name: 'Terraform'
+
     runs-on: ubuntu-latest
```

Comment:
Does this mean latest LTS?

Reply:
No.  It's just a loosely specified Ubuntu environment for simple installs and shell commands